{'description': {'en': 'Gentoo cross-compilation overlay for embedded devices'},
 'feed': ['https://cgit.gentoo.org/proj/embedded-cross.git/atom/'],
 'homepage': 'https://cgit.gentoo.org/proj/embedded-cross.git/',
 'name': 'embedded-cross',
 'owner': [{'email': 'armin76@gentoo.org',
            'name': u'Ra\xfal Porcel',
            'type': 'project'}],
 'quality': 'experimental',
 'source': [{'type': 'git',
             'uri': 'git://anongit.gentoo.org/proj/embedded-cross.git'},
            {'type': 'git',
             'uri': 'https://anongit.gentoo.org/git/proj/embedded-cross.git'},
            {'type': 'git',
             'uri': 'git+ssh://git@git.gentoo.org/proj/embedded-cross.git'}],
 'status': 'official'}
$ pmaint sync embedded-cross
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
Already up-to-date.
*** syncing embedded-cross
*** synced embedded-cross
 * Sync succeeded
$ pmaint regen -t 2 embedded-cross
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING: 'use_enable' function invoked by x11-libs/e_dbus-0.5.0.063 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/e_dbus-0.5.0.061 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/e_dbus-0.5.0.062 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.


!!! ERROR: app-editors/vim-core-7.1.330 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'

!!! ERROR: app-editors/vim-core-7.2.442 failed.
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '207' 'depend'

!!! ERROR: app-editors/vim-7.3 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.


!!! ERROR: x11-libs/gtk+-2.14.5 failed.
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.330.ebuild'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '207' 'depend'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.330.ebuild'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '160' 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.330.ebuild'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: app-editors/vim-core-7.0.235 failed.
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!        vim-core-7.1.330.ebuild, line   5:  Called inherit 'vim'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '176' 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.442.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim/vim-7.3.ebuild'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.442.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: app-editors/vim-core-7.2.303 failed.
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim/vim-7.3.ebuild'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '207' 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.442.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim/vim-7.3.ebuild'
!!!        vim-core-7.2.442.ebuild, line   6:  Called inherit 'vim'

!!! ERROR: x11-drivers/xf86-video-glamo-9999 failed.
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'

!!! ERROR: app-editors/vim-core-7.1.123 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:

!!! ERROR: app-editors/vim-core-7.0.243 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                 vim-7.3.ebuild, line   7:  Called inherit 'vim'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.235.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
 * 
 * "/home/mgorny/repos/embedded-cross/sci-geosciences/gpsd/gpsd-2.37.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!             gtk+-2.14.5.ebuild, line   7:  Called inherit 'gnome.org' 'flag-o-matic' 'eutils' 'libtool' 'virtualx'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'

!!! ERROR: app-editors/vim-core-7.3 failed.
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '207' 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.235.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: x11-drivers/xf86-video-glamo-20100626 failed.
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'virtualx'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '207' 'depend'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/virtualx.eclass'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.235.ebuild'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '228' 'depend'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '207' 'depend'
!!!                 ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'

!!! ERROR: x11-drivers/xf86-input-tslib-0.0.5 failed.
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!        vim-core-7.0.235.ebuild, line   5:  Called inherit 'vim'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/virtualx.eclass'

!!! ERROR: app-misc/mdbus-9999 failed.
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.303.ebuild'
!!!                 ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                virtualx.eclass, line  38:  Called die
!!! The die message:
!!!  virtualx eclass require EAPI=2 or newer.

!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.123.ebuild'
!!!              ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.243.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.303.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '187' 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.243.ebuild'
!!!                 ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '248' 'depend'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!              ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.303.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.123.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.243.ebuild'

!!! ERROR: x11-terms/vala-terminal-1.1.1 failed.
!!!                 ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!              ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '237' 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.123.ebuild'

!!! ERROR: x11-wm/enlightenment-0.16.999.063 failed.
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.3.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '165' 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!        vim-core-7.0.243.ebuild, line   5:  Called inherit 'vim'
!!!                         ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'

!!! ERROR: x11-base/xglamo-20081205-r2 failed.
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!        vim-core-7.2.303.ebuild, line   6:  Called inherit 'vim'
!!!              ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!        vim-core-7.1.123.ebuild, line   5:  Called inherit 'vim'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                  ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                         ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!   xf86-video-glamo-9999.ebuild, line  10:  Called inherit 'x-modular'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.3.ebuild'
!!!                      ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-input-tslib/xf86-input-tslib-0.0.5.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 189' for eclass vim

received unknown com: export CATEGORY='dev-util' PR=r0 PVR='0.23' PV='0.23' EBUILD='/home/mgorny/repos/embedded-cross/dev-util/pkgconfig/pkgconfig-0.23.ebuild' P='pkgconfig-0.23' PF='pkgconfig-0.23' PN=pkgconfigfailed
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
Exception in thread Thread-13:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                  ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'

!!! ERROR: sys-mobilephone/apm2-2.0.0 failed.
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.3.ebuild'
!!!             isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '218' 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!                      ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-input-tslib/xf86-input-tslib-0.0.5.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
 * 
 * "/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.1.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!                  ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '245' 'depend'
!!!   xf86-video-glamo-20100626.ebuild, line  16:  Called inherit 'x-modular'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!            vim-core-7.3.ebuild, line   7:  Called inherit 'vim'
!!!          isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-input-tslib/xf86-input-tslib-0.0.5.ebuild'

!!! ERROR: sys-mobilephone/freesmartphone-framework-0.5.1 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:

!!! ERROR: dev-lang/lua-5.1.4-r5 failed.
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '201' 'depend'
!!!                  ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'

!!! ERROR: sys-mobilephone/fso-raw-1.0.0 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!       ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!   xf86-input-tslib-0.0.5.ebuild, line   7:  Called inherit 'toolchain-funcs' 'eutils' 'linux-info' 'x-modular'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                          ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.063.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 188' for eclass vim

!!!                      ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
received unknown com: export CATEGORY='dev-lang' PR=r3 PVR='2.5.4-r3' PV='2.5.4' EBUILD='/home/mgorny/repos/embedded-cross/dev-lang/python/python-2.5.4-r3.ebuild' P='python-2.5.4' PF='python-2.5.4-r3' PN=pythonfailed
Exception in thread Thread-10:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

 * 
 * "/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.1.330
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.2.442
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-7.3
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.0.235
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-editors/vim-core-7.2.303
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: dev-util/pkgconfig-0.23
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.1.123
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-drivers/xf86-video-glamo-9999
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.0.243
!!! caught exception Metadata Exception: pkg ebuild src: sci-geosciences/fso-gpsd-0.8, attr eapi
error: parsed eapi doesn't match sourced eapi while processing ebuild src: sci-geosciences/fso-gpsd-0.8
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-editors/vim-core-7.3
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: dev-lang/python-2.5.4-r3
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-drivers/xf86-input-tslib-0.0.5
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-drivers/xf86-video-glamo-20100626
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: app-misc/mdbus-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-libs/gtk+-2.14.5
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sci-geosciences/tangogps-0.9.6
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-terms/vala-terminal-1.1.1
!!! caught exception Metadata Exception: pkg ebuild src: x11-wm/enlightenment-0.16.999.063, attr eapi
error: parsed eapi doesn't match sourced eapi while processing ebuild src: x11-wm/enlightenment-0.16.999.063
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-base/xglamo-20081205-r2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/lua-5.1.4-r5
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-util/strace-4.5.15
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/freesmartphone-framework-9999
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/apm2-2.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-libs/gnutls-2.8.5
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/freesmartphone-framework-0.5.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/python-2.6.2-r1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/fsodeviced-0.9.4
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: 
!!! ERROR: sys-mobilephone/freesmartphone-framework-0.5 failed.
!!!                  ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!                          ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.063.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '183' 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!               x-modular.eclass, line  44:  Called inherit 'eutils' 'libtool' 'multilib' 'toolchain-funcs' 'flag-o-matic' 'autotools' 'git'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-base/xglamo/xglamo-20081205-r2.ebuild'
!!!    ebuild-default-functions.lib, line 230:  Called __internal_inherit 'linux-info'
!!!                      ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!              mdbus-9999.ebuild, line   5:  Called inherit 'git'
!!!             isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!              isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.063.ebuild'
!!!                    ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!               ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 188' for eclass linux-info

received unknown com: export CATEGORY='dev-lang' PR=r1 PVR='2.6.4-r1' PV='2.6.4' EBUILD='/home/mgorny/repos/embedded-cross/dev-lang/python/python-2.6.4-r1.ebuild' P='python-2.6.4' PF='python-2.6.4-r1' PN=pythonpath
!!!                      ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '285' 'depend'
Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                   x-modular.eclass, line  44:  Called inherit 'eutils' 'libtool' 'multilib' 'toolchain-funcs' 'flag-o-matic' 'autotools' 'git'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '173' 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-base/xglamo/xglamo-20081205-r2.ebuild'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '200' 'depend'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!   enlightenment-0.16.999.063.ebuild, line   8:  Called inherit 'enlightenment'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                      ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                    ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!     vala-terminal-1.1.1.ebuild, line   5:  Called inherit 'git' 'autotools'

!!! ERROR: x11-wm/enlightenment-0.16.999.061-r1 failed.
!!!        ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment'
!!!       ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                              ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.1.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/apm2/apm2-2.0.0.ebuild'
!!!                   ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 215' for eclass enlightenment

received unknown com: export CATEGORY='x11-base' PR=r6 PVR='1.5.3-r6' PV='1.5.3' EBUILD='/home/mgorny/repos/embedded-cross/x11-base/xorg-server/xorg-server-1.5.3-r6.ebuild' P='xorg-server-1.5.3' PF='xorg-server-1.5.3-r6' PN='xorg-server'path
Exception in thread Thread-4:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                  ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-base/xglamo/xglamo-20081205-r2.ebuild'

!!! ERROR: dev-lang/vala-0.8.1.1 failed.
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/apm2/apm2-2.0.0.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                    ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '275' 'depend'
!!!                              ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.1.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/apm2/apm2-2.0.0.ebuild'
!!!                     ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!      xglamo-20081205-r2.ebuild, line   5:  Called inherit 'autotools' 'git' 'toolchain-funcs'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-raw/fso-raw-1.0.0.ebuild'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
 * 
 * "/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20090704.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                    ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-lang/lua/lua-5.1.4-r5.ebuild'

!!! ERROR: sys-mobilephone/freesmartphone-framework-20090704 failed.
!!!                  isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.1.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!              apm2-2.0.0.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                     ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: x11-terms/vala-terminal-1.2 failed.
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: sys-mobilephone/fsotdld-0.3.0 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-lang/lua/lua-5.1.4-r5.ebuild'
!!! Call stack:
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-raw/fso-raw-1.0.0.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:

!!! ERROR: sys-mobilephone/libframeworkd-glib-20090614 failed.
!!!                            ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!   freesmartphone-framework-0.5.1.ebuild, line   6:  Called inherit 'git'
!!!                         ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '254' 'depend'

!!! ERROR: sys-mobilephone/freesmartphone-framework-0.5.5 failed.
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-raw/fso-raw-1.0.0.ebuild'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '179' 'depend'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                            ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/dev-lang/lua/lua-5.1.4-r5.ebuild'
!!!                   ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'

!!! ERROR: sys-mobilephone/fso-alsa-1.0.0 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                         ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!            ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!           fso-raw-1.0.0.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                      ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                       ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.ebuild'
!!!                   ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!            lua-5.1.4-r5.ebuild, line   7:  Called inherit 'autotools' 'eutils' 'multilib' 'portability' 'toolchain-funcs' 'versionator'
!!!                     ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '198' 'depend'

!!! ERROR: sys-mobilephone/fso-specs-2010.05.11.2 failed.
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'base'
!!!                         ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '296' 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-lang/vala/vala-0.8.1.1.ebuild'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '208' 'depend'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 203' for eclass base

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.1.0' PV='0.1.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodatad/fsodatad-0.1.0.ebuild' P='fsodatad-0.1.0' PF='fsodatad-0.1.0' PN=fsodatadpath
Exception in thread Thread-5:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                             ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.061-r1.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'portability'
!!!                   ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '266' 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                         ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!   freesmartphone-framework-0.5.ebuild, line   6:  Called inherit 'git'
!!!              ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-lang/vala/vala-0.8.1.1.ebuild'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 296' for eclass portability

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20091021 PV=20091021 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20091021.ebuild' P='freesmartphone-framework-20091021' PF='freesmartphone-framework-20091021' PN='freesmartphone-framework'path
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
Exception in thread Thread-6:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                      ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                             ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.061-r1.ebuild'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '205' 'depend'
!!!                   ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!              ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/dev-lang/vala/vala-0.8.1.1.ebuild'
!!!                 isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.061-r1.ebuild'
!!!                      ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '285' 'depend'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!          ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                                 ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20090704.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!!                           ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090614.ebuild'
!!!                                 ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20090704.ebuild'
!!!                     ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-alsa/fso-alsa-1.0.0.ebuild'
!!!                           ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090614.ebuild'
!!!              ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '245' 'depend'

!!! ERROR: sys-mobilephone/fsoinitd-0.1.0 failed.
!!!   enlightenment-0.16.999.061-r1.ebuild, line   8:  Called inherit 'enlightenment'
!!!            vala-0.8.1.1.ebuild, line   7:  Called inherit 'gnome2'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!!                      ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-alsa/fso-alsa-1.0.0.ebuild'
!!!               isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090614.ebuild'
!!!                     isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20090704.ebuild'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'gnome2'
!!!                              ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-alsa/fso-alsa-1.0.0.ebuild'
!!!              ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!           ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!!   libframeworkd-glib-20090614.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!                              ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
!!!                      ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 208' for eclass enlightenment

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.9.4' PV='0.9.4' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fsousaged/fsousaged-0.9.4.ebuild' P='fsousaged-0.9.4' PF='fsousaged-0.9.4' PN=fsousagedpath
Exception in thread Thread-8:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 266' for eclass gnome2

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090227 PV=20090227 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild' P='libframeworkd-glib-20090227' PF='libframeworkd-glib-20090227' PN='libframeworkd-glib'path
!!!                  isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
Exception in thread Thread-16:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   freesmartphone-framework-20090704.ebuild, line   6:  Called inherit 'git'
!!!           fsotdld-0.3.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!          fso-alsa-1.0.0.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!   freesmartphone-framework-0.5.5.ebuild, line   5:  Called inherit 'distutils'
!!!         ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!       vala-terminal-1.2.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '203' 'depend'
!!!               ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                      ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-specs/fso-specs-2010.05.11.2.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'base'
!!!                    ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 306' for eclass git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090614 PV=20090614 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui-efl/libframeworkd-phonegui-efl-20090614.ebuild' P='libframeworkd-phonegui-efl-20090614' PF='libframeworkd-phonegui-efl-20090614' PN='libframeworkd-phonegui-efl'failed
Exception in thread Thread-7:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!                          ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                      ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-specs/fso-specs-2010.05.11.2.ebuild'

!!! ERROR: sys-mobilephone/libframeworkd-glib-99999999 failed.
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 286' for eclass base

!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=99999999 PV=99999999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-99999999.ebuild' P='libframeworkd-phonegui-99999999' PF='libframeworkd-phonegui-99999999' PN='libframeworkd-phonegui'path
sys-mobilephone/freesmartphone-framework-0.5
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/python-2.6.4-r1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/fsomusicd-0.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-wm/enlightenment-0.16.999.061-r1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-libs/gnutls-2.8.6
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-base/xorg-server-1.5.3-r6
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/vala-0.8.1.1
!!! caught exception Metadata Exception: pkg ebuild src: sys-mobilephone/gsm0710muxd-0.9.3.1, attr eapi
error: parsed eapi doesn't match sourced eapi while processing ebuild src: sys-mobilephone/gsm0710muxd-0.9.3.1
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/freesmartphone-framework-20090704
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-terms/vala-terminal-1.2
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-efl-20090506
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fsotdld-0.3.0
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-efl-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/freesmartphone-framework-0.5.5
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fso-alsa-1.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fso-specs-2010.05.11.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsodatad-0.1.0
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libfsobasics-0.9.9
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/freesmartphone-framework-20091021
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsoinitd-0.1.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsousaged-0.9.4
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libgsm0710-1.2.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-20090227
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libphone-ui-shr-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-20090614
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libphone-utils-20090724
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/liException in thread Thread-9:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!            ebuild-default-functions.lib, line 230:  Called __internal_inherit 'distutils'
!!!                       ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 286' for eclass distutils

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090724 PV=20090724 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090724.ebuild' P='libframeworkd-phonegui-20090724' PF='libframeworkd-phonegui-20090724' PN='libframeworkd-phonegui'path
Exception in thread Thread-12:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsoinitd/fsoinitd-0.1.0.ebuild'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!          isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-specs/fso-specs-2010.05.11.2.ebuild'
!!!  unknown inherit command from pythonic side, 'gen_metadata 306' for eclass git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090506 PV=20090506 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui-efl/libframeworkd-phonegui-efl-20090506.ebuild' P='libframeworkd-phonegui-efl-20090506' PF='libframeworkd-phonegui-efl-20090506' PN='libframeworkd-phonegui-efl'failed
Exception in thread Thread-11:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                   ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!   fso-specs-2010.05.11.2.ebuild, line   6:  Called inherit 'base' 'autotools' 'git'
!!!                   ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsoinitd/fsoinitd-0.1.0.ebuild'
!!!    ebuild-default-functions.lib, line 230:  Called __internal_inherit 'base'
!!!                   ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '266' 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsoinitd/fsoinitd-0.1.0.ebuild'
!!!               ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 255' for eclass base

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='2010.05.11.2' PV='2010.05.11.2' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libfso-glib/libfso-glib-2010.05.11.2.ebuild' P='libfso-glib-2010.05.11.2' PF='libfso-glib-2010.05.11.2' PN='libfso-glib'path
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!          cornucopia_git.eclass, line   6:  Called inherit 'base' 'git' 'autotools'
!!!          fsoinitd-0.1.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!                   ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 238' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.9.6' PV='0.9.6' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsotransport/libfsotransport-0.9.6.ebuild' P='libfsotransport-0.9.6' PF='libfsotransport-0.9.6' PN=libfsotransportpath

!!! ERROR: sys-mobilephone/libframeworkd-phonegui-20090506 failed.
Exception in thread Thread-15:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                           ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-99999999.ebuild'
!!!                       ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!                           ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-99999999.ebuild'
!!!                       ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!               isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-99999999.ebuild'
!!!                       ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '286' 'depend'
!!!                       ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!   libframeworkd-glib-99999999.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!                               ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090506.ebuild'
!!!         ebuild-default-functions.lib, line 230:  Called __internal_inherit 'base'
!!!                               ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090506.ebuild'
!!!                    ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 193' for eclass base

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='2.0.1' PV='2.0.1' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/mdbus2/mdbus2-2.0.1.ebuild' P='mdbus2-2.0.1' PF='mdbus2-2.0.1' PN=mdbus2path
!!!                   isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090506.ebuild'
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   libframeworkd-phonegui-20090506.ebuild, line  21:  Called inherit 'shr'
!!!             ebuild-default-functions.lib, line 230:  Called __internal_inherit 'shr'
!!!                        ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 219' for eclass shr

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=99999999 PV=99999999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/phonefsod/phonefsod-99999999.ebuild' P='phonefsod-99999999' PF='phonefsod-99999999' PN=phonefsodpath
Exception in thread Thread-14:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

bframeworkd-phonegui-99999999
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/mioctl-0.1.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-20090724
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/mterm2-1.9.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-efl-20090724
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libfso-glib-2010.05.11.2
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/ophonekitd-20090724
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-20090506
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libfsotransport-0.9.6
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/phoneui-apps-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/mdbus2-2.0.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/pyphonelog-20090901
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/phonefsod-99999999
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/serial_forward-1.2
Error running an operation:
 Unhandled exception in regen_cache
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7f1c240d2838> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7f1c240d2730> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7f1c240d2628> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7f1c240d2520> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7f1c240d2418> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7f1c240d2310> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7f1c240d2208> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7f1c240d2100> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7f1c2412afc8> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7f1c2412aec0> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7f1c2412adb8> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7f1c2412acb0> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7f1c2412aba8> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7f1c2412aaa0> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7f1c2412a998> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7f1c2412a310> ignored
 * Cache regen failed with 246
